### DISCUSSION TITLE
Vcpkg SSL Support

### DISCUSSION THREAD

**Discussion by Darkratos** [uNetworking/uWebSockets#1819] [Show and tell]
When installing the vcpkg version of uWebSockets, it will not install SSL support of uSockets.
This is not stated anywhere in the documentation, but it should be there.
To enable it, install usockets[ssl] or, in manifest mode, add:
`{
  "name": "usockets",
  "features": ["ssl"]
}`
to the dependencies.

**Comment by uNetworkingAB** [comment]
We don't manage vcpkg distribution, that's Microsoft. Ask them.
